regparser/notice/xml.py

```diff
@@ -41,6 +42,100 @@ def _set_date_attr(self, date_type, value):
             value = value.isoformat()
         dates_tag.attrib["eregs-{}-date".format(date_type)] = value
 
+    def derive_agencies(self, agencies=[]):
```

Comment:
Real quick, the reason for that is that the mutable value is instantiated at class _construction_ time, rather than _instantiation_ time. In other words, the "default" value is actually a static variable of the class.